### PR TITLE
TorchBlade benchmarks based on TorchBench

### PR DESCRIPTION
**Pull request by zzpmiracle**
Run pytorch benchmark based on torchdynamo, as mentioned in [#450](https://github.com/alibaba/BladeDISC/issues/450)

### CODE REVIEW

File: pytorch_blade/benchmark/TorchBench/torchbench.py [hunk 1]
```diff
@@ -0,0 +1,326 @@
+#!/usr/bin/env python
+import gc
```
Comment (fortianyou) [inline pytorch_blade/benchmark/TorchBench/torchbench.py:2]: Is this file modified?

File: pytorch_blade/benchmark/TorchBench/common.py [hunk 2]
```diff
@@ -0,0 +1,1313 @@
...
+
+            results.append(experiment(model, example_inputs))
+            print(" ".join(map(str, results)))
+
+
+def help(fn):
+    return fn.__doc__
+
+
+def parse_args():
```
Comment (fortianyou) [inline pytorch_blade/benchmark/TorchBench/common.py:733]: Nit: how about overriding the parse_args from `torchdynamo.benchmarks.common` directly? Such as writing the following codes in some files like bladebench.py
```python
import logging
from torchdynamo.benchmarks import common
from torchdynamo.benchmarks import torchbench

def parse_args():
    ...
    return args

if __name__ == "__main__":
    common.parse_args = parse_args
    logging.basicConfig(level=logging.WARNING)
    warnings.filterwarnings("ignore")
    commons.main(torchbench.TorchBenchmarkRunner(), torchbench.original_dir)
    
```

Comment (fortianyou) [reply pytorch_blade/benchmark/TorchBench/common.py:733]: And then call the TorchBench with the following command:

`python bladebench.py --speedup-blade-disc -d cuda --isolate --float32 --skip-accuracy-check 2>&1 | tee speedup_blade.log`

**Comment by fortianyou** [comment]
Overall look good to me, but I would like to copy as less codes as possible from torchdynamo.

File: pytorch_blade/benchmark/TorchBench/test_torch_bench.sh [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+# limitations under the License.
+
+# !/bin/bash
+# install dependencies
+apt install -y git git-lfs libglib2.0-0 libsndfile1 && pip install librosa torchdynamo torchvision torchaudio torchtext --extra-index-url https://download.pytorch.org/whl/cu113
+
+script_dir=$(cd $(dirname "$0"); pwd)
+pushd $script_dir
+# setup for torchbenchmark
+git clone https://github.com/jansel/benchmark.git --recursive torchbenchmark
```
[the remark this answers is not in the document]
Comment (zzpmiracle) [reply pytorch_blade/benchmark/TorchBench/test_torch_bench.sh:19]: This is forked for torchdynamo specially, removed or modified some models.
We can use official repository for most model.

File: pytorch_blade/benchmark/TorchBench/torchbench.py (hunk 1 above)
Comment (zzpmiracle) [reply pytorch_blade/benchmark/TorchBench/torchbench.py:2]: Not changed, but not include in torchdynamo wheel.

File: pytorch_blade/benchmark/TorchBench/test_torch_bench.sh (hunk 3 above)
[the remark this answers is not in the document]
Comment (fortianyou) [reply pytorch_blade/benchmark/TorchBench/test_torch_bench.sh:19]: Then I prefer to use the official repo, and skip the set of models for TorchDynamo. And can we benchmark all models for TorchScript?

File: pytorch_blade/benchmark/TorchBench/torchbench.py (hunk 1 above)
Comment (fortianyou) [reply pytorch_blade/benchmark/TorchBench/torchbench.py:2]: So we can remove this file as discussed offline?

**Review by fortianyou — APPROVED** [review]
LGTM

**Comment by zzpmiracle** [comment]
OK, I have remove all copied code, and remove "WIP" from this PR’s title
